db/migrate/20170212055707_create_teams.rb

```diff
@@ -0,0 +1,16 @@
+class CreateTeams < ActiveRecord::Migration[5.0]
+  def change
+    create_table :teams do |t|
+      t.string :team_name
```

Comment:
yeah, `first_name`, `last_name` and `slack_name` sounds good. no need for an extra table right now :)